### ISSUE TITLE
Could u push the tabs repo?

### ISSUE THREAD

**Issue by jackchoumine** [krhoyt/Toodles#14]
hello I come from your articl [Building with Stencil: Tabs](https://ionicframework.com/blog/building-with-stencil-tabs/)  I am implementing tabs followed thsi artilce, But I wannt your source code.  I find the repo link is 404  in the bottom of article, Could u push it  again?

**Comment by krhoyt** [comment]
Yeah, I had to change my repo structure due to being laid off at Ionic. You can email me directly parkerkrhoyt@gmail.com and I will send you an archive containing the code.
